playground/main.ts

```diff
@@ -167,6 +175,8 @@ const renderPlaygroundWidget = (options: WidgetOptions & { assertNoEnglishLeaks?
   if (customize) {
     addAfterTransformHooks(signIn as OktaSignInAPIV3);
   }
+  // promise resolves when widget ready event is received
+  return readyPromise;
```

Comment:
Goal here is to only proceed after widget is ready to avoid trying to run assertions on the widget before it completely finishes bootstrapping.